lib/core/test/bench/db/Main.hs

```diff
@@ -244,24 +252,54 @@ bgroupSeqState db = bgroup "SeqState"
 --
 -- - 50 inputs
 -- - 100 outputs
-bgroupTxHistory :: DBLayerBench -> Benchmark
-bgroupTxHistory db = bgroup "TxHistory"
-    --           #NBatch  #BatchSize #NInputs #NOutputs
-    [ bTxHistory       1         100        1        1
-    , bTxHistory       1        1000        1        1
-    , bTxHistory      10          10        1        1
-    , bTxHistory     100          10        1        1
-    , bTxHistory       1         100       10       10
-    , bTxHistory       1        1000       10       10
-    , bTxHistory       1       10000       10       10
-    , bTxHistory       1         100       50      100
-    , bTxHistory       1        1000       50      100
-    , bTxHistory       1       10000       50      100
+bgroupWriteTxHistory :: DBLayerBench -> Benchmark
+bgroupWriteTxHistory db = bgroup "TxHistory (Write)"
+    --              #NBatch  #BatchSize #NInputs #NOutputs  #SlotRange
+    [ bTxHistory          1         100        1        1     [1..100]
+    , bTxHistory          1        1000        1        1     [1..100]
+    , bTxHistory         10          10        1        1     [1..100]
+    , bTxHistory        100          10        1        1     [1..100]
+    , bTxHistory          1         100       10       10     [1..100]
+    , bTxHistory          1        1000       10       10    [1..1000]
+    , bTxHistory          1       10000       10       10   [1..10000]
+    , bTxHistory          1         100       50      100     [1..100]
+    , bTxHistory          1        1000       50      100     [1..100]
```

Comment:
This one seems to be taking quite a long time compared with the rest (like 90ms per tx, and 1000 txs).
Perhaps instead of batch sizes [100, 1000], we could do [50, 100, 200], for these big transactions.